### DISCUSSION TITLE
With Geo tags, can lightweight MMM do media contribution percentage, ROI, and optimization by each Geo?

### DISCUSSION THREAD

**Discussion by DongHarry-Kang** [google/lightweight_mmm#269] [Q&A]
In the dummy data, when I ran date with Geo tags, the media contribution percentage, ROI, and optimization are shown as 1 number. However, can we extract media contribution percentage, ROI, and optimization by each Geo?

**Comment by statm3n** [comment]
- If geo level model is built, [get_posterior_metrics](https://github.com/google/lightweight_mmm/blob/8152c43c8b52eb8c020c1c54c7e4bfa5140e42d0/lightweight_mmm/lightweight_mmm.py#L559) returns media contribution and roi by (sample x channel x geo)
- Geo level optimization? to my knowledge, not feasible in lmmm yet. You are more than welcome to hack and explore the implementation

**Reply by DongHarry-Kang** [reply]
Thank you very much!
